doc/README.txt

```diff
@@ -1,18 +1,20 @@
+.. highlight:: sh
+
 Matplotlib documentation
 ========================
 
-
```

Comment:
This seems to duplicate documenting_mpl.rst  Can all of this just be removed and replaced by reference?

Reply:
yup, fixed